### PR TITLE
SPCTR-3335: Fix - Separator height unit setting to px from % on refresh of editor

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Separator height '%' unit set to default 'px' after refresh.
https://brainstormforce.atlassian.net/browse/SPCTR-3335

### Screenshots

https://u.pcloud.link/publink/show?code=XZoND3VZi9ybRoLKYOp2saenWRf9LhGRFWOy

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,6 +226,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Forms - Resolved an issue wherein the Forms block would not work inside Containers in Astra's Custom Layout.
 * Fix: Global - Removed the Block Editor Spacing setting for FSE themes.
 * Fix: Image - Resolved an issue in which masked images would get cut-off when the zoom-in hover effect was used.
+* Fix: Separator - Resolved an issue where the unit of height is setting to pixel from Percentage on refresh of editor.
```
Comment (bsf-zanev) [inline README.md:229]:
```suggestion
* Fix: Separator - Resolved an issue wherein the height would revert from percentage to pixels after reloading the editor.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update needed.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

File: src/blocks/separator/attributes.js [hunk 2]
```diff
@@ -152,7 +152,6 @@ const attributes = {
 		},
 	},
 	separatorHeightType: {
-		type: 'number',
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply src/blocks/separator/attributes.js:155]: Added type: 'string',
done backward compatibility test.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**